openforcefields/tests/test_water_models.py

```diff
@@ -160,7 +211,7 @@ def test_tip5p(water_molecule):
 
 @pytest.mark.parametrize(
     "water_model,pattern",
-    [("tip3p", "^tip3p(?!.*fb)"), ("tip3p_fb", "^tip3p_fb"), ("opc", "^opc")],
+    [("tip3p", "^tip3p(?!.*fb)"), ("tip3p_fb", "^tip3p_fb"), ("tip4p_fb", "^tip4p_fb"), ("opc3", "^opc"), ("opc", "^opc(?!.*3)")],
```

Comment:
```suggestion
    [("tip3p", "^tip3p(?!.*fb)"), ("tip3p_fb", "^tip3p_fb"), ("tip4p_fb", "^tip4p_fb"), ("opc3", "^opc3"), ("opc", "^opc(?!.*3)")],
```
If this pattern is mean to hit all files starting with `opc3`, I think that needs to be explicit; the logic of the other pattern hitting "start with `opc` but don't be `opc3`" won't be accounted for in these cases since they're split out into different runs of this test which don't interact with each other.

If I'm right, I'd have expected a test to fail, though?